src/main/java/io/weaviate/client/v1/graphql/query/argument/Bm25Argument.java

```diff
@@ -28,7 +30,37 @@ public String build() {
     if (properties != null) {
       arg.add(String.format("properties:%s", Serializer.arrayWithQuotes(properties)));
     }
+    if (searchOperator != null) {
+      arg.add(String.format("searchOperator:%s", searchOperator.build()));
+    }
 
     return String.format("bm25:{%s}", String.join(" ", arg));
   }
+
+  @AllArgsConstructor(access = AccessLevel.PRIVATE)
+  public static class SearchOperator implements Argument {
+    private static final String OR = "Or";
+    private static final String AND = "And";
+
+    private String operator;
+    private int minimumMatch;
+
+    public static SearchOperator and() {
+      return new SearchOperator(AND, 0); // minimumMatch ignored for And
+    }
+
+    public static SearchOperator or(int minimumMatch) {
+      return new SearchOperator(OR, minimumMatch);
+    }
+
+    @Override
+    public String build() {
+      String query = "{operator:" + operator;
+      if (operator != AND) {
```

Comment:
it doesn't make sense adding `minimumOrTokensMatch` in case of `OR` right?

Reply:
Weaviate ignores `minimumOrTokensMatch` argument if the operator is `OR`